recipes/suitesparse/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_Mac.mk" SuiteSparse_config/SuiteSparse_config.mk
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_linux.mk" SuiteSparse_config/SuiteSparse_config.mk
+fi
+
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export INSTALL_LIB="${PREFIX}/lib"
+export INSTALL_INCLUDE="${PREFIX}/include"
+
+export BLAS="-lopenblas"
+export LAPACK="-lopenblas"
```

Comment:
The following additions would enable TBB for SPQR:

export SPQR_CONFIG="${SPQR_CONFIG} -DHAVE_TBB"
export TBB="-ltbb"_

If desired to enable OpenMP (used by CHOLMOD?), it looks like the addition would be:
export CFLAGS="$CFLAGS -fopenmp"